django/thunderstore/api/cyberstorm/views/team.py

```diff
@@ -52,6 +62,58 @@ def get_queryset(self) -> QuerySet[TeamMember]:
         )
 
 
+class CyberstormTeamAddMemberRequestSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
+    role = serializers.ChoiceField(choices=TeamMember._meta.get_field("role").choices)
+
+
+class CyberstormTeamAddMemberResponseSerialiazer(serializers.Serializer):
+    user = serializers.CharField(max_length=User._meta.get_field("username").max_length)
+    role = serializers.ChoiceField(choices=TeamMember._meta.get_field("role").choices)
+    team = serializers.CharField(max_length=Team._meta.get_field("name").max_length)
+
+
+class AddTeamMemberAPIView(APIView):
+    @swagger_auto_schema(
+        request_body=CyberstormTeamAddMemberRequestSerialiazer,
+        responses={200: CyberstormTeamAddMemberResponseSerialiazer},
+        operation_id="cyberstorm.team.members.add",
+        tags=["cyberstorm"],
+    )
+    def post(self, request, team_name, format=None):
+        try:
+            team = Team.objects.get(name=team_name)
+        except Team.DoesNotExist:
+            return Response(
+                json.dumps(
+                    {
+                        "error": "Team not found",
+                    }
+                ),
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        serializer = CyberstormTeamAddMemberRequestSerialiazer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        form = AddTeamMemberForm(
+            user=request.user,
+            data={
+                "team": team.pk,
+                "user": serializer.validated_data["user"],
+                "role": serializer.validated_data["role"],
+            },
+        )
+
+        if form.is_valid():
+            team_member = form.save()
+            return Response(
+                CyberstormTeamAddMemberResponseSerialiazer(team_member).data,
+                status=status.HTTP_200_OK,
```

Comment:
You _can_ omit the status line entirely since 200 is the default status. Explicit is better than implicit so it's not necessary, but in this case it probably lets you fit this on a single line.